### ISSUE TITLE
alt tag

### ISSUE THREAD

**Issue by mplusb** [MachoThemes/simple-author-box#92]
I noticed that there is no ALT text set to author images. I manually uploaded a custom image to be used for a specific user but the plugin does not use the ALT text inputted to the image.

Can you fix this in future updates?

**Comment by giucu91** [comment]
Fixed. See https://github.com/MachoThemes/simple-author-box/commit/635f7792c15e904d64416132dfa2927a20c00236
